guides/anchor/2-bridge-server.md

```diff
@@ -37,8 +37,8 @@ code="USD"
 issuer="GAIUIQNMSXTTR4TGZETSQCGBTIF32G2L5P4AML4LFTMTHKM44UHIN6XQ"
 
 [database]
-type = "mysql"  # or "postgres" if you created a postgres database
-url = "dbuser:dbpassword@/stellar_bridge"
+type = "postgres"
+url = "postgres://root@localhost/bridge?sslmode=disable"
```

Comment:
We should use some dummy values rather than encourage hosting the db on localhost, with the root user, or with sslmode disabled. We should use the postgres scheme in the URL like you've changed to though, so I'd run with something like:

```suggestion
url = "postgres://dbuser@dbhost/bridge"
```